tutorandroid/templates/android/build/config/tutor.yaml

```diff
@@ -68,7 +68,10 @@ BRANCH:
   ALTERNATE_HOST: ''
 
 #Platform names
-PLATFORM_NAME: "{{ PLATFORM_NAME }}"
+# TODO: Temporarily set PLATFORM_NAME based on OPENEDX_COMMON_VERSION to 'OpenEdx' for compatibility with the Android app theme directory
+# (https://github.com/openedx/openedx-app-android expects 'OpenEdx'). This is addressed in PR https://github.com/openedx/openedx-app-android/pull/335,
```

Comment:
> (https://github.com/openedx/openedx-app-android expects 'OpenEdx')

Do you have a link that this is the value it expects? The actual issue is that platform name has spaces which cause issues with themeDirs. Therefore, the name is hardcoded to a value with no spaces.

Reply:
This [PR](https://github.com/openedx/openedx-app-android/pull/335)  basically ensures theme works properly  without being dependent on the PLATFORM_NAME value, which previously caused issues with theme directories.